### PR TITLE
static check resolved

### PR DESCRIPTION
**Pull request by gunjanmimo**
Signed-off-by: gunjanmimo <gunjan.mtbpaul@gmail.com>

### CODE REVIEW

**Comment by gunjanmimo** [comment]
@girishranganathan @leecalcote sir, All the errors are gone.  😃😃

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
       uses: actions/setup-go@v1
       with:
         go-version: ${{ secrets.GO_VERSION }}
-    - run: GOPROXY=direct GOSUMDB=off GO111MODULE=on go get -u github.com/kisielk/errcheck; /home/runner/go/bin/errcheck -tags draft ./...
+    - run: GOPROXY=direct GOSUMDB=off GO111MODULE=on go build
```
Comment (nveenjain) [inline .github/workflows/ci.yml]: Hi @gunjanmimo , I think this line needs to be unedited, we certainly need to check all the errors that can occur at runtime beforehand... This command disables those error checks...

File: meshes/meshops.pb.go [hunk 2]
```diff
@@ -3,12 +3,15 @@
```
Comment (nveenjain) [inline meshes/meshops.pb.go]: I'm not sure whether this file should be modified or not... 
Due to the comment:-
```go
// Code generated by protoc-gen-go. DO NOT EDIT.
```
I guess static checks would still pass if this file along with supported_ops.go were not modified...

**Comment by nveenjain** [comment]
Like when we call
```golang
defer yamlDecoder.Close()
```
For example in https://github.com/layer5io/meshery-istio/blob/master/istio/istio.go#L754
We are using yamlDecoder.Close(), that may result in an error, you can resolve these error checks using this guide:-
https://blog.learngoprogramming.com/5-gotchas-of-defer-in-go-golang-part-iii-36a1ab3d6ef1

**Comment by leecalcote** [comment]
@gunjanmimo Ping when you've had a chance to revisit here. 👍

**Comment by nveenjain** [comment]
@leecacote , I guess this issue got resolved in #65

**Comment by leecalcote** [comment]
@nveenjain Oh, ok. I'll go ahead and close this issue.

**Pull request closed without merging**